packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -96,9 +99,11 @@ def test_vip(dcos_api_session, container: marathon.Container, vip_net: marathon.
     proxy container that will ping the origin container VIP and then assert
     that the expected origin app UUID was returned
     '''
-    failure_stack = []
-    for test in setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net):
-        cmd, origin_app, proxy_app, proxy_net = test
+    errors = 0
+    tests = setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net)
+    for vip, hosts, cmd, origin_app, proxy_app in tests:
+        log.info("Testing :: VIP: {}, Hosts: {}".format(vip, hosts))
```

Comment:
I was wishing for info logs here while trying to figure out which mesos task maps to the failed test. This is great.